test/libyul/yulInterpreterTests/side_effect_free.yul

```diff
@@ -0,0 +1,18 @@
+{
+	// These can be removed by the optimizer and should not
+	// appear in the trace.
+	pop(gas())
+	pop(extcodesize(0))
+	pop(extcodehash(0))
+	pop(returndatasize())
+	pop(sload(0))
+	pop(pc())
+	pop(msize())
+        pop(mload(0))
+        pop(sload(0))
+        pop(msize())
```

Comment:
Could make use of `pop(keccak256(0,0))` here.

Reply:
and of whitespace alignment...